db/seeds.rb

```diff
@@ -5,4 +5,12 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
-User.create!(username:  "Example User", password: "password", password_confirmation: "password")
+# User.create!(username:  "Example User", password: "password", password_confirmation: "password")
+
+@doctor1 = Doctor.create!(name: 'Doctor1', specialization: 'eye specialist', available_times: 'monday-friday', hourly_rate: 250, picture: ('https://asset.cloudinary.com/dcphbr3ka/607f68c350ebf6725e2804f73eba01bf'))
+
+
+@doctor2 = Doctor.create!(name: 'Doctor2', specialization: 'eye specialist', available_times: 'monday-friday', hourly_rate: 250, picture: ('https://asset.cloudinary.com/dcphbr3ka/87e6862dcd2ddc1f58c293f7c999c534'))
+
+@doctor3 = Doctor.create!(name: 'Doctor3', specialization: 'eye specialist', available_times: 'monday-friday', hourly_rate: 250, picture: ('https://asset.cloudinary.com/dcphbr3ka/6ca49db87b0e6d8ed3ba05f84cf43472'))
+
```

Comment:
Hello @OybekKayumov, can we put actual names here? These data would be represented on the front end of the app and it would be inappropriate to have `doctor 1` as the name. You can add any random name just make sure they are full names.